README.markdown

```diff
@@ -21,67 +21,47 @@ It should work on any system that supports Google's [Cloud SDK](https://develope
 
 In order to use this module, you will need to
 [signup](https://developers.google.com/compute/docs/signup)
- for a Google Cloud Platform account and enable Google Compute Engine.
+for a Google Cloud Platform account and enable Google Compute Engine.
 
-You will also need to designate one machine to be your Puppet Device Agent.
-This machine will be responsible for provisioning objects into Google Compute
-using the `gcutil` command-line utility that is now bundled as part of the Cloud
-SDK. Follow the setup instructions for the
-[Google Cloud SDK](https://developers.google.com/cloud/sdk/) and make sure
-to authenticate as instructed.
-
-Next, create your `device.conf` file on the Agent. The default location for
-this file can be discovered by running the command (typically
-`/etc/puppet/device.conf`):
+### Setup a Puppet Agent with Google Cloud SDK
```

Comment:
Great, thanks.  Is "controller machine" a better term for it?  It's technically not a puppet master, but it is deploying resources.